.github/maintainers_guide.md

```diff
@@ -84,6 +84,32 @@ Currently we apply all recommended rules.
 
 The list of format options is defined in the `deno.jsonc` file. They closely resemble the default values.
 
+### Update latest SDK dependencies
+
+Check if the Deno related [slack samples](slack-samples) reference the latest versions of their dependencies.
+
+This usually involves updating multiple repositories ([dependency graph](#dependency-graph)), so take a deep breath, relax, and take your time as you run through the following steps, working backwards from the template dependencies up to the [slack samples](slack-samples) themselves:
+
+* There are two **main developer-facing Deno SDK dependencies** that samples rely on:
```

Comment:
I don't think it is, this is all context for the Release Commander regarding updating samples and not the SDK. IMO rather than moving these instructions, the runbook should just be updated to reflect the new way we're releasing. The Release Commander should still have context from the runbook on the different libraries that are necessary for samples, it just doesn't need to go into doing the library releases themselves.

The actual instructions for how to do those releases should be a part of the Releasing section of the maintainers guide in that repo. I'm not seeing much context here that we'd want to interweave into the existing Releasing section.